### DISCUSSION TITLE
References to 'Military Force' in some definitions should be 'Military Personnel Force' (curated in EventOntology)

### DISCUSSION THREAD

**Discussion by jonathanvajda** [CommonCoreOntology/CommonCoreOntologies#840] [Announcements]
In the Event Ontology, the following terms refer to a _Military Force_, but there is no such term precisely:

- cco:ActofMilitaryForce def:"An Intentional Act of employing a Military Force to achieve some desired result."
- cco:ActofMilitaryService def:"An Act of Employment wherein a Person serves as a member of a Military Force, whether voluntarily or by conscription."
- cco:CarrierAirWing def:"A Military Force that is trained and equipped to conduct air operations while embarked on an aircraft carrier." [the immediate parent is cco:MilitaryPersonnelForce]

The closest relevant term is cco:MilitaryPersonnelForce. My guess, is that 'MilitaryForce' was original and ambiguous, but when it was disambiguated the other references weren't updated? Whatever the cause, it seems that the definitions for the above (bulleted terms)  could get some adjustments.

I suggest the following changes (bold are additions):
- cco:ActofMilitaryForce def:"An Intentional Act of employing a Military **Personnel** Force to achieve some desired result."
- cco:ActofMilitaryService def:"An Act of Employment wherein a Person serves as a member of a Military **Personnel** Force, whether voluntarily or by conscription."
- cco:CarrierAirWing def:"A Military **Personnel** Force that is trained and equipped to conduct air operations while embarked on an aircraft carrier."

Alternatively, give 'Military Force' as an alternative_label for MilitaryPersonnelForce?

**Comment by cameronmore** [comment]
Seems like a good thing to fix, I would rather opt for adding 'Military Force' as an alternative label for MilitaryPersonnelForce, as people often speak about Forces in such a way.

**Comment by mark-jensen** [comment]
@cameronmore  There is too much work to be done here on the definitions for 'armed force' and its subs. Adding this alt label doesn't help. Closing the PR until improvements are made

**Comment by mark-jensen** [comment]
An attempt is made to distinguish paramilitary from military, but it is done badly. 

from def for 'paramilitary force': "...similar to those of a professional **Military** [note cap, but no 'military' class exists in CCO], as which is not included as part of a Government's formal Armed Forces"

Yet, 'paramilitary force' is a type of 'armed force' as is 'military personnel farce', so, well, that's just wrong.

From [Military](https://en.wikipedia.org/wiki/Military) wiki:
> In broad usage, the terms "armed forces" and "military" are often synonymous, although in technical usage a distinction is sometimes made in which a country's armed forces may include other [paramilitary](https://en.wikipedia.org/wiki/Paramilitary) forces such as armed police.

So, the def's use of 'formal' is doing all the work really. But, it doesn't help one understand what the underlying distinction may be.

**Reply by jonathanvajda** [reply]
I recommend we consider the classes out of scope for the common core.

Since there are known, actively developed military ontologies that deal with this domain explicitly, it seems right to demote it from Core. I propose that this happens even before negotiating with those developers who are actively rescoping and refactoring their own.

What argument remains, that these belong in the Core?

**Comment by PhiBabs935** [comment]
@cameronmore  @jonathanvajda 

'Military Force' is a term that exists in the CCO Military Operations Ontology, and it is not the same thing as a Military Personnel Force, rather it is the sort of thing that includes an MPF as part:

An Object Aggregate that consists of Military Personnel, Military Resources (such as weapon systems or equipment), and necessary support, or some combination thereof.

Derived from 

(DOD) 1. An aggregation of military personnel, weapon systems, equipment, and necessary support, or combination thereof. 
> definition of 'force' in JP 1

**Comment by PhiBabs935** [comment]
@APCox , do you have anything to add here, as you curated the Military Operation Ontology
